File: Programming Basic/Conditional_statements_exercise/08_Scholarship.py
import math

income = float(input())
grade = float(input())
min_salary = float(input())

scholarship = min_salary * 0.35
excellent_scholarship = grade * 25

if income < min_salary and grade > 4.50:
      if scholarship > excellent_scholarship:
            print(f'You get a Social scholarship {math.floor(scholarship)} BGN')
      elif grade > 5.50:
            print(f'You get a scholarship for excellent results {math.floor(excellent_scholarship)} BGN')
else:
      print('You cannot get a scholarship!')







# if income < min_salary and grade > 4.5 or grade < 5.5:
#       print(f'You get a Social scholarship {math.floor(stipendia)} BGN')
# elif grade >= 5.5:
#       print(f'You get a scholarship for excellent results {math.floor(excellent_scholarship)} BGN')
# else:
#       print('You cannot get a scholarship!')

File: Java Script Basic - April 2020/Conditional_Statements_Exercise/08_Scholarship.js
function scholarship(income, grade, min_salary) {
    income = parseFloat(income)
    grade = parseFloat(grade)
    min_salary = parseFloat(min_salary)

    let scholarship1 = min_salary * 0.35
    let excellent_scholarship = grade * 25

    if (income < min_salary && grade > 4.5 && grade < 5.5) {
        console.log(`You get a Social scholarship ${Math.floor(scholarship1)} BGN'`)
    } else if (grade >= 5.5) {
        console.log(`You get a scholarship for excellent results ${Math.floor(excellent_scholarship)} BGN`)
    } else {
        console.log('You cannot get a scholarship!')
    }

}

scholarship(300, 5.65, 420)